### PR TITLE
Bug 1429570 - Change 'Bugzilla eMail' to 'Mozillians Identity Provide…

### PR DESCRIPTION
**Pull request by MichaelKohler**
…r e-mail'

### CODE REVIEW

**Comment by MichaelKohler** [comment]
@akatsoulas r?

File: docs/profiles.rst [hunk 1]
```diff
@@ -19,7 +19,7 @@ We are collecting the following information for each Rep:
  * Display Name
  * Mozillians Profile
  * Private Email
- * Bugzilla Email
+ * Mozillians Identity Provider e-mail
```
Comment (akatsoulas) [inline docs/profiles.rst]: The term  `Mozillians Identity Provider` may be confused with an Identity Provider service which provides identities to users. Maybe we should replace this with something that communicates that we mean the identity that is linked in mozillians.org. Unless you mean something completely different here. Thoughts?

Comment (MichaelKohler) [reply docs/profiles.rst]: Do you mean for this instance here only or generally?

Henrik and I came up with that name and though it was specific enough and didn't have any other idea.

Comment (hmitsch) [reply docs/profiles.rst]: Had a conversation with @akatsoulas ... 
We believe that the term `Mozilla Login Identity` would currently be the best functional description.

-Henrik

Comment (MichaelKohler) [reply docs/profiles.rst]: OK; will change it. Thanks!

**Comment by MichaelKohler** [comment]
@akatsoulas rebased to master and changed to the new wording

**Pull request merged**